src/GitVersion.Configuration/GitVersionConfiguration.cs

```diff
@@ -124,6 +124,14 @@ private string GetVersionInBranchPattern()
     [JsonPropertyDefault(DefaultSemanticVersionFormat)]
     public SemanticVersionFormat SemanticVersionFormat { get; internal set; }
 
+    [JsonIgnore]
+    VersionStrategies IGitVersionConfiguration.VersionStrategy => VersionStrategies.Length == 0
+        ? VersionCalculation.VersionStrategies.None : VersionStrategies.Aggregate((one, another) => one | another);
+
+    [JsonPropertyName("strategies")]
+    [JsonPropertyDescription($"Specifies which version strategies (one or more) will be used to determine the next version. Following values are available: 'ConfigNextVersion', 'MergeMessage', 'TaggedCommit', 'TrackReleaseBranches', 'VersionInBranchName' and 'TrunkBased'.")]
+    public VersionStrategies[] VersionStrategies { get; internal set; } = [];
```

Comment:
I don't understand how it works that `VersionStrategies` is an array where but not in the `IGitVersionConfiguration` interface.

Reply:
![image](https://github.com/GitTools/GitVersion/assets/56404113/a243edb5-0184-48d3-8d4a-24fff1bd7b26)

From IGitVersionConfiguration.VersionStrategy you call VersionStrategies property.

Reply:
Ah, I didn't notice that. Okay, that's fine for now. But I would like to see the split between the core domain model and DTO used for de/serialization in a future PR. I might contribute that, if time allows.